Makefile

```diff
@@ -542,6 +544,11 @@ upload-preloaded-images-tar: out/minikube # Upload the preloaded images for olde
 push-storage-provisioner-image: storage-provisioner-image ## Push storage-provisioner docker image using gcloud
 	gcloud docker -- push $(STORAGE_PROVISIONER_IMAGE)
 
+.PHONY: push-kic-base-image
+push-kic-base-image: kic-base-image ## Push kic-base docker image using gcloud
+	(docker pull $(KIC_BASE_IMAGE) && (echo "Image already exist"; exit 1) || echo "Image doesn't exist in registry. Pushing...")
+	gcloud docker -- push $(KIC_BASE_IMAGE)
```

Comment:
lets add a prompt too, "Are you sure you want to push ?" if they answer "y" it willl push

Reply:
```
(docker pull gcr.io/k8s-minikube/kicbase:v0.0.10-aaa && (echo "Image already exist"; exit 1) || echo "Image doesn't exist in registry")
Error response from daemon: manifest for gcr.io/k8s-minikube/kicbase:v0.0.10-aaa not found: manifest unknown: Failed to fetch "v0.0.10-aaa" from request "/v2/k8s-minikube/kicbase/manifests/v0.0.10-aaa".
Image doesn't exist in registry
⚠️  'Are you sure you want to push gcr.io/k8s-minikube/kicbase:v0.0.10 ?'
Do you want to proceed? (Y/N): n
make: *** [Makefile:561: push-kic-base-image] Error 1
```

Reply:
added lines 136-146 with required logic